source/Octopus.Tentacle.Tests.Integration/ScriptServiceV3AlphaIntegrationTest.cs

```diff
@@ -18,7 +18,7 @@ namespace Octopus.Tentacle.Tests.Integration
     public class ScriptServiceV3AlphaIntegrationTest : IntegrationTest
     {
         [Test]
-        [TentacleConfigurations(testCurrentVersionOnly: true)]
+        [TentacleConfigurations(specificServiceToTest: typeof(IAsyncClientScriptServiceV3Alpha))]
```

Comment:
As this is a Type does it leave room for the engineer to get it wrong? Could we make it an enum along with the properties in `TentacleConfigurationsAttribute` and other places this flows to? 

That way you can't get it wrong e.g. pass a random, unrelated type

Reply:
`public Type ScriptServiceToTest { get; }`

Reply:
This was updated in e8483cef